src/test/java/fr/lunatech/timekeeper/services/UserEventServiceTest.java

```diff
@@ -0,0 +1,133 @@
+package fr.lunatech.timekeeper.services;
+
+import fr.lunatech.timekeeper.models.time.UserEvent;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserEventServiceTest {
+
+    UserEventService userEventService = new UserEventService();
+
+    @Test
+    void shouldBeTrueForEventWithDurationUnderAWeekDuration() {
```

Comment:
I don't understand the purpose of this test. Maybe the name should be changed or add a comment